### DISCUSSION TITLE
How to display LATEX format with ctk?

### DISCUSSION THREAD

**Discussion by dan-adi** [TomSchimansky/CustomTkinter#1705] [General]
I want to insert LATEX text in textbox or label. Is there a way to do it? What I tried doesn't display the latex format ...

**Comment by Akascape** [comment]
@dan-adi Is it something like this?: https://www.tutorialspoint.com/how-to-display-latex-in-real-time-in-a-text-box-in-tkinter

**Reply by ghost** [reply]
I think using `matplotlib` takes away from the beautiful-ness (not a word??) of `customtkinter`'s modern UI.

**Reply by abhaymakes** [reply]
You could also make the background color dark, which can match with customtkinter.

**Comment by abhaymakes** [comment]
Or you could refer to [this](https://youtu.be/4MROGBovKC0) video tutorial, implementing it in Customtkinter and Tkinter would be fairly easy.
